deployment/postgres-init-db/sql/tables/merlin/external_events/plan_derivation_group.sql

```diff
@@ -0,0 +1,25 @@
+create table merlin.plan_derivation_group (
+    plan_id integer not null,
+    derivation_group_name text not null,
+    last_acknowledged_at timestamp with time zone default now() not null,
+
+    constraint plan_derivation_group_pkey
+      primary key (plan_id, derivation_group_name),
+    constraint plan_derivation_group_references_plan_id
+      foreign key (plan_id)
+      references merlin.plan(id),
+    constraint plan_derivation_group_references_derivation_group_name
```

Comment:
Please shorten this constraint name to `pdg_derivation_group_exists`. Postgres truncates constraint names past a certain length.